plugins/friends-exporter

```diff
@@ -1,2 +1,2 @@
 repository=https://github.com/MarbleTurtle/FriendsExporter.git
-commit=6596ac1981833e22caf38fa0d67fe456f3932c08
+commit=06248b3e399ce72c5ce0fe475aa32fc843571988
```

Comment:
As I saw your discussion with MarbleTurtle in the discord, you'll want to add an `authors` field so the bot doesn't mislabel your PRs.

```suggestion
commit=06248b3e399ce72c5ce0fe475aa32fc843571988
authors=MarbleTurtle,Raitab
```